src/main/java/htsjdk/samtools/util/IntervalList.java

```diff
@@ -739,6 +739,59 @@ public static IntervalList difference(final Collection<IntervalList> lists1, fin
                 subtract(lists2, lists1));
     }
 
+    /**
+     * A utility function for finding the intervals in the first list that have at least 1bp overlap with any interval
+     * in the second list.
+     *
+     * @param lhs the first collection of IntervalLists
+     * @param lhs the second collection of IntervalLists
+     * @return an IntervalList comprising of all intervals in the first IntervalList that have at least 1bp overlap with
+     * any interval in the second.
+     */
+    public static IntervalList overlaps(final IntervalList lhs, final IntervalList rhs) {
+        return overlaps(Collections.singletonList(lhs), Collections.singletonList(rhs));
+    }
+
+    /**
+     * A utility function for finding the intervals in the first list that have at least 1bp overlap with any interval
+     * in the second list.
+     *
+     * @param lists1 the first collection of IntervalLists
+     * @param lists2 the second collection of IntervalLists
+     * @return an IntervalList comprising of all intervals in the first collection of lists that have at least 1bp
+     * overlap with any interval in the second lists.
+     */
+    public static IntervalList overlaps(final Collection<IntervalList> lists1, final Collection<IntervalList> lists2) {
+        final SAMFileHeader header = lists1.iterator().next().getHeader().clone();
```

Comment:
If you look at the rest of the "actions", you'll see either they return `null` (`intersection`) or throw an exception (`concatenate`).  And then some of the other actions (ex. `union`) depend on these.  So I think it's fine the way it is (albeit I need to add a check for an empty list).